go.mod

```diff
@@ -1,5 +1,17 @@
 module github.com/kumojin/go-echo-mixed-middleware
 
-require github.com/labstack/echo/v4 v4.6.1
+require github.com/labstack/echo/v4 v4.9.1
 
-go 1.15
+require (
+	github.com/labstack/gommon v0.4.0 // indirect
+	github.com/mattn/go-colorable v0.1.13 // indirect
+	github.com/mattn/go-isatty v0.0.16 // indirect
+	github.com/valyala/bytebufferpool v1.0.0 // indirect
+	github.com/valyala/fasttemplate v1.2.2 // indirect
+	golang.org/x/crypto v0.1.0 // indirect
+	golang.org/x/net v0.1.0 // indirect
+	golang.org/x/sys v0.1.0 // indirect
+	golang.org/x/text v0.4.0 // indirect
+)
+
+go 1.17
```

Comment:
If the minimal version of go is 1.16, it should be 1.16 here too

Reply:
I was inspired by what Echo does. They put `go 1.17` (see https://github.com/labstack/echo/pull/2049) to benefit new from new Go features but still test up to 1.16 (https://github.com/labstack/echo/blob/master/.github/workflows/echo.yml#L31).

However I'm still trying to figure out why the golangcli-lint fails (maybe this https://github.com/golangci/golangci-lint/issues/2649 but still need to run some tests to be sure).

Reply:
In the end I used `go 1.17` here and in the CI. I never managed to run the CI with go 1.16, golangcli-lint always fails with this version and the dependency bump.

go 1.17 is the same version than [echo 4.9.1](https://github.com/labstack/echo/blob/v4.9.1/go.mod) so I think we're good here.